Classes/PostProcessor/PublishStatusPostProcessor.php

```diff
@@ -32,8 +32,20 @@ public function process($response): array
     {
         $response = (array)$response;
         foreach ($response as $key => $item) {
+            // Skip _root entry
+            if ($key === '_root') {
+                continue;
+            }
+
+            // Check if leistung is a leistung or a zustaendigkeit with a leistung as array item
```

Comment:
`Check if leistung is a leistung or a zustaendigkeit with a leistung` Auahhh. Hoffentlich wird das nie public :D